### PR TITLE
Update auth api instructions

### PR DESCRIPTION
**Pull request by jkeiser**
Just documenting my journey running the auth api locally.

### CODE REVIEW

File: bin/auth-api/README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ instances.
 While working on the auth stack, we still need to run it locally and configure things to point to our local auth stack:
 
 - update auth-api env vars in `bin/auth-api/.env.local`
+    - if you don't have .env.local yet, copy the `auth api local dev .env.local` key into the `.env.local` file.
```
Comment (stack72) [inline bin/auth-api/README.md:39]: I don't understand what this sentence means

File: bin/auth-api/README.md [hunk 2]
```diff
@@ -44,12 +45,14 @@ While working on the auth stack, we still need to run it locally and configure t
     VITE_AUTH_API_URL=http://localhost:9001
     VITE_AUTH_PORTAL_URL=http://localhost:9000
   ```
-- run the backend but using the local auth stack by setting env var `SI_AUTH_API_URL=http://localhost:9001` (
-  ex: `SI_AUTH_API_URL=http://localhost:9001 buck2 run dev:up`)
-- run the db migrations (`pnpm run db:reset`) locally after booting your local database
+- run the db migrations (`pnpm run db:reset`) locally after booting your local database (run `buck2 dev:stop` and then `buck2 dev` and hope nobody connects in the meantime!).
```
Comment (stack72) [inline bin/auth-api/README.md:48]: "nobody connects in the meantime"?

**Review by stack72 — COMMENTED** [review]
2 clarifying questions

File: bin/auth-api/README.md (hunk 2 above)
Comment (jkeiser) [reply bin/auth-api/README.md:48]: db:reset gets angry if anyone else is using Postgres.

**Review by stack72 — APPROVED** [review]

**Comment by stack72** [comment]
This can be closed as it was merged as part of #5143

**Pull request closed without merging**